components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/bindings/handlers/TokenBindingExpiryEventHandler.java

```diff
@@ -78,6 +86,11 @@ public void handleEvent(Event event) throws IdentityEventException {
                 .EventProperty.CONTEXT);
         try {
             if (request == null) {
+                if (log.isDebugEnabled()) {
+                    log.debug("Request object is null. Hence getting the session related information from event and "
```

Comment:
Mention as either Servlet or HTTP Request :)
People might get confused with Request object in OIDC